### ISSUE TITLE
Is this how Finder windows are supposed to act?

### ISSUE THREAD

**Issue by mepelman** [ttscoff/bunch#163]
A bunch command to close all Finder windows,
```
Finder
- XX
```
proceeds to open user's home directory window after execution. 

v. 1.4.6 (132)

**Comment by ttscoff** [comment]
Yes, because the process of activating Finder automatically brings that 
window up automatically if no other window is open. Open to hacking on a 
feature request if there's enough interest in making this scenario just 
close all Finder windows without activating in the process.

Based on my recent feature request fulfillments, I need somewhere 
between 1 and 3 upvotes to get the ball rolling... ;)

-Brett
